src/server/api/mutations/updateServiceManager.js

```diff
@@ -1,4 +1,3 @@
-import { GraphQLError } from "graphql/error";
 import { getConfig } from "../../../server/api/lib/config";
```

Comment:
It looks like getConfig isn't being used in this file either.  Should that be deleted also?

Reply:
Yes, done! Good catch!

Reply:
There are lots of places like this to clean up -- unused imports and other lints. I'm interested in doing that or maybe someone else could do it. I'm using vscode with integrated linting and I like to look at clean files (personal preference) but it's also a good practice to have zero lints.